File: pyg-groupck/pyg-manager-web/src/main/webapp/js/controller/loginController.js
//定义控制器
app.controller("loginController", function ($scope,loginService) {


    //定义查询所有方法,所有的方法都必须绑定在$scope作用域
    $scope.showName = function () {
        //使用内置服务$http向后台发送ajax请求
        loginService.showLoginName().success(function (data) {
            $scope.loginName = data.loginName;
        })
    };
})

File: pyg-groupck/pyg-order-web/src/main/java/com/pyg/order/controller/LoginController.java
package com.pyg.order.controller;


import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import javax.servlet.http.HttpServletRequest;
import java.util.HashMap;
import java.util.Map;

/**
 * Created by on 2018/8/16.
 */
@RestController
@RequestMapping("/login")
public class LoginController {

    /**
     * 需求：获取用户登录名，进行回显
     */
    @RequestMapping("showName")
    public Map showLoginName(HttpServletRequest request){
        
        //获取用户登录名
        String loginName = request.getRemoteUser();
        //创建map封装登录名
        Map loginMap = new HashMap();
        loginMap.put("loginName",loginName);

        return loginMap;

    }

}
